app/views/wiki/_header.html.erb

```diff
@@ -31,7 +31,13 @@
 
     <div style="padding-top:8px;text-align:center">
       <span class="d-none d-lg-inline">
-        <span rel="tooltip" title="The number of authors for this page." data-placement="top"><i style="color:#888;" class="fa fa-user"></i> <%= number_with_delimiter(@node.authors.size) %></span>  |
+        <span rel="popover" style='cursor: pointer;' title="Authors" data-html="true" data-placement="top" data-content= "
+        <ul>
+        <% @node.authors.each do |author| %>
+          <li style='list-style: none;'><a href='/profile/<%= author.username %>' style='color: #009bff;'>@<%= author.username %></a></li>
+        <% end %>
+      </ul>
+      "> <i style="color:#888;" class="fa fa-user"></i> <%= number_with_delimiter(@node.authors.size) %></span>  | 
```

Comment:
Revert these changes basically.

Reply:
I thought they were changed. Anyways I will change them now. Thanks

Reply:
@gaurav2699 Hey, I have reverted the changes, I hope this might work. Sorry about this. It's a very simple contribution and its taking so time.

Reply:
That's OK @Priyaraj17 it's always a learning process and we appreciate your time too!